hatchways/team-gingerbread
# Integration of Edit Profile

Modified profile/edit route to modify user Profile with information received in the body of POST request

### What this PR does (required):
- allows the user to modify their own profile
- currently only working via Postman, getting TypeError (failed to fetch) in the app

### Screenshots / Videos (front-end only):
-Postman Screenshot
![image](https://user-images.githubusercontent.com/78042694/142968457-150a6362-275c-42fa-bbea-fa37d79a3b36.png)

### Any information needed to test this feature (required):
- connect your MongoDB database to the app by specifying MONGO_URI to the link to your database in the .env file
- run server in node
- open Postman, send a post request to /profile/edit with information in the body
- body must contain the id of the user whose profile you would like to update as well as the properties you would like to modify
- after sending the request, the server will send back the updated object (can open MongoDB Cloud to confirm data has been updated)

### Any issues with the current functionality (optional):
- this feature only works via Postman but is not functional when using the app to submit the POST request
![image](https://user-images.githubusercontent.com/78042694/142968827-a91b6811-1ba9-42a5-ba67-78babb9e58f6.png)
- still having issues with the token validation
